azurerm/resource_arm_storage_account.go

```diff
@@ -205,7 +206,7 @@ func resourceArmStorageAccount() *schema.Resource {
 
 						"default_action": {
 							Type:     schema.TypeString,
-							Optional: true,
+							Required: true,
```

Comment:
Is this not a breaking change?

Reply:
The title says "changing default to allow" and the change is to make it required? 🤔

Reply:
🤦‍♂ my bad.

This is currently set to Required in the documentation: https://www.terraform.io/docs/providers/azurerm/r/storage_account.html#default_action - as such this PR updates it to match.. I'll update the title.

Reply:
I'll defer to you and @katbyte on whether you want to call this a breaking change or not.

Reply:
given this block is now Computed (and the API returns a default response here) I think we're  safe to not call this a breaking change here